app/views/articles/tags/_sidebar.html.erb

```diff
@@ -63,7 +63,11 @@
       <% end %>
       <div class="sidebar-data">
         <div>
-          <%= pluralize Article.published.cached_tagged_with(@tag).size, "Post" %> Published
+          <% if @tag_model.requires_approval? %>
+            <%= pluralize Article.published.cached_tagged_by_approval_with(@tag).size, "Post" %> Published
+          <% else %>
+            <%= pluralize Article.published.cached_tagged_with(@tag).size, "Post" %> Published
+          <% end %>
```

Comment:
It might also be worth it to surface these two queries. For example (haven't tested it), in `stories_controller#handle_tag_index` we call:

```ruby
if @tag_model.requires_approval?
  @num_published_articles = Article.published.cached_tagged_by_approval_with(@tag).size
else
  @num_published_articles = Article.published.cached_tagged_with(@tag).size
end
```

and then in this partial you render that variable